yarn-project/aztec.js/src/contract/sent_tx.ts

```diff
@@ -60,10 +62,10 @@ export class SentTx {
    */
   public async wait(opts?: WaitOpts): Promise<FieldsOf<TxReceipt>> {
     if (opts?.debug && opts.waitForNotesSync === false) {
-      throw new Error('Cannot set getNotes to true if waitForNotesSync is false');
+      throw new Error('Cannot set debug to true if waitForNotesSync is false');
     }
     const receipt = await this.waitForReceipt(opts);
-    if (receipt.status !== TxStatus.MINED && receipt.status !== TxStatus.REVERTED) {
+    if (!(receipt.status === TxStatus.MINED || (receipt.status === TxStatus.REVERTED && opts?.dontThrowOnRevert))) {
```

Comment:
I'd probably consider making this behaviour the default :)

Reply:
Yep, `dontThrowOnRevert` defaults to false, so it throws on reverts by default. Is that what you meant?